### PR TITLE
Split README into section files

### LINKED ISSUE #102: Move away from the monolithic README.md

### PR DESCRIPTION
**Pull request by ruippeixotog**
Fixes #102.

### CODE REVIEW

**Review by leifwickland — APPROVED** [review]
It appears to me you faithfully moved all the expected sections to their own pages.

File: core/docs/overrride-behaviour-for-sealed-families.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+## Override behaviour for sealed families
```
Comment (melrief) [inline core/docs/overrride-behaviour-for-sealed-families.md]: Too many `r` in the name of this file :smile:

**Review by melrief — CHANGES_REQUESTED** [review]
The link to overrride behaviour for sealed families is broken, everything else LGTM

File: core/docs/overrride-behaviour-for-sealed-families.md (hunk 1 above)
Comment (ruippeixotog) [reply core/docs/overrride-behaviour-for-sealed-families.md]: Ahah, nice catch! :D Fixed in fee4be3.

**Review by melrief — APPROVED** [review]

**Pull request merged**